Doc/library/imaplib.rst

```diff
@@ -204,7 +204,23 @@ An :class:`IMAP4` instance has the following methods:
 
 .. method:: IMAP4.append(mailbox, flags, date_time, message)
 
-   Append *message* to named mailbox.
+   Append *message* to a named *mailbox*.
+
+   *flags* is a space-separated string containing IMAP flags tokens.
+   System flags defined in :rfc:`3501` are:
+
+   ``\Seen`` - Message has been read.
+   ``\Answered`` - Message has been answered.
+   ``\Flagged`` - Message is "flagged" for urgent/special attention.
+   ``\Deleted`` - Message is "deleted" for removal by later ``EXPUNGE``.
+   ``\Draft`` - Message has not completed composition (marked as a draft).
+   ``\Recent`` - Message is "recently" arrived in this mailbox.
+
```

Comment:
Please keep the bullet points. And we can also link directly to the relevant RFC section:


```suggestion

   System flags defined in :rfc:`RFC 3501 <3501#section-2.3.2>` are:

   * ``\Seen`` - Message has been read.
   * ``\Answered`` - Message has been answered.
   * ``\Flagged`` - Message is "flagged" for urgent/special attention.
   * ``\Deleted`` - Message is "deleted" for removal by later ``EXPUNGE``.
   * ``\Draft`` - Message has not completed composition (marked as a draft).
   * ``\Recent`` - Message is "recently" arrived in this mailbox.
```


So it looks more like:

<img width="786" alt="image" src="https://github.com/python/cpython/assets/1324225/54ebf69d-0ef3-439a-9c57-781299fa9c89">

Instead of:

<img width="809" alt="image" src="https://github.com/python/cpython/assets/1324225/5574c44a-647b-4a52-b875-7e8372bd194e">